### ISSUE TITLE
`blink`

### ISSUE THREAD

**Issue by Appsurdgames** [Alphish/gm-community-toolbox#104]
For blinking of a cursor, we want a script to return true/false periodically.

Here is a possible implementation:

`/// @function blink([offTime, onTime, phaseShift, clock])
/// @param {real}	offTime			time during which false (default: 500 ms)
/// @param {real}	onTime			time during which true (default: same as offTime)
/// @param {real}	phaseShift		offset time (default: 0)
/// @param {real}	clock			which clock to use (default: current_time)
///
/// @description	For a cursor, be able to made it blink

function blink(offTime = 500, onTime = offTime, phaseShift = 0, clock = current_time)
{
	return (clock + phaseShift) mod (offTime + onTime) >= offTime;
}

**Comment by Alphish** [comment]
I'm not quite sure about using name `blink`, as it's so short people may use it for other variables. But I think the functionality might be a good addition. In that regard, it's similar to the `wave` suggestion.

**Comment by Alphish** [comment]
Potential name suggestion: `cycle_blink`, which might also match `cycle_around` and `cycle_between` for waving functions, if those names are accepted.

**Comment by Appsurdgames** [comment]
I agree that the current name is too short/generic. Your suggestion `cycle_blink` is fine with me
